### PR TITLE
avoid diving 0 by zero and missing buyers page button

### PR DESCRIPTION
**Pull request by yspringfield**
Avoid showing NAN as result of division in percentage fields.
Bildschirmfoto 2022-06-02 um 8 55 47 AM

Fixed https://github.com/blockframes/blockframes/issues/8512#issuecomment-1134610130 : missing `View all buyers' activies` button on dashboard.

### CODE REVIEW

File: apps/festival/festival/src/app/dashboard/home/home.component.scss [hunk 1]
```diff
@@ -70,3 +70,12 @@ analytics-map h3{
...
 }
+
+.insights-wrapper{
+  flex-direction: column;
+  display: flex;
+  button{
+    align-self: center;
+    margin-top: 16px;
+  }
+}
```
Comment (RemcoSimonides) [inline apps/festival/festival/src/app/dashboard/home/home.component.scss:81]: Change class name of `.see_titles` button and use same css as that one:
```
  .see_titles {
    margin: 24px auto auto auto;
  }
```

File: apps/festival/festival/src/app/dashboard/analytics/title/title-analytics.component.ts [hunk 2]
```diff
@@ -56,17 +59,17 @@ function toScreenerCards(screeningRequests: Analytics<'title'>[], invitations: P
...
     {
       title: 'Acceptation Rate',
-      value: `${acceptationRate}%`,
+      value: !invitations.length ? '-' : `${acceptationRate}%`,
       icon: 'sentiment_satisfied'
     },
     {
       title: 'Traction Rate',
-      value: `${traction}%`,
+      value: !invitations.length ? '-' : `${traction}%`,
```
Comment (RemcoSimonides) [inline apps/festival/festival/src/app/dashboard/analytics/title/title-analytics.component.ts:72]: ```value: invitations.length ? `${participationRate}%` : '-',```

File: apps/festival/festival/src/app/dashboard/home/home.component.scss [hunk 3]
```diff
@@ -62,11 +58,16 @@ analytics-map.carousel-map {
...
   display: flex;
   align-items: center;
 }
 
-.insights-map{
+.margin-bottom {
   margin-bottom: 24px;
 }
+
+.see_titles {
```
Comment (RemcoSimonides) [inline apps/festival/festival/src/app/dashboard/home/home.component.scss:70]: Please change the name of the class as this name isn't applicable anymore for "View all buyers' activities"

File: apps/festival/festival/src/app/dashboard/home/home.component.html [hunk 4]
```diff
@@ -125,6 +126,8 @@ <h3>Most active buyers</h3>
               {{ askingPriceRequested }}
             </ng-template>
           </bf-table>
+
+          <button mat-button class="see_titles" routerLink="/c/o/dashboard/home/buyer">View All Buyers' Activity</button>
```
Comment (RemcoSimonides) [inline apps/festival/festival/src/app/dashboard/home/home.component.html:130]:
```suggestion
          <a mat-button class="see_titles" routerLink="buyer">View All Buyers' Activity</a>
```

File: apps/festival/festival/src/app/dashboard/home/home.component.scss [hunk 5]
```diff
@@ -62,11 +58,16 @@ analytics-map.carousel-map {
...
+  display: block;
+  margin: auto;
+}
```
Comment (RemcoSimonides) [inline apps/festival/festival/src/app/dashboard/home/home.component.scss:73]: I just noticed that this css is below a 
```
@include layout-bp(xs) {
  ...
}
```
Please move it above as these layout-bp's should be at the bottom of the css file

**Review by RemcoSimonides — APPROVED** [review]

File: apps/festival/festival/src/app/dashboard/home/home.component.scss [hunk 6]
```diff
@@ -56,17 +52,23 @@ analytics-map.carousel-map {
...
 
-.insights-map{
+.margin-bottom {
   margin-bottom: 24px;
 }
+
+.link_buttons {
+  display: block;
+  margin: auto;
+  width: min-content;
```
[the remark this answers is not in the document]
Comment (yspringfield) [reply apps/festival/festival/src/app/dashboard/home/home.component.scss]: The `View Buyers' Activities` extends the whole width of it's parent element regardless of margin auto.
setting width to `min/fit -content` resolves it.

**Pull request merged**